pyproject.toml

```diff
@@ -28,6 +28,7 @@ dependencies = [
     'sentence_transformers >= 2',
     'sentencepiece>=0.1.95',
     'tomli; python_version < "3.11"',
+    'tokenizers >= 0.13.2; python_version >= "3.11"',
```

Comment:
Can you add a link to this PR or the tokenizers release notes?

Reply:
done!